packages/actor-query-operation-path-one-or-more/lib/ActorQueryOperationPathOneOrMore.ts

```diff
@@ -72,22 +117,21 @@ export class ActorQueryOperationPathOneOrMore extends ActorAbstractPath {
           path.graph,
         ),
       });
-    } else {
-      // If (!sVar && !oVar)
-      const blankNode = this.generateBlankNode();
-      const bString = termToString(blankNode);
-      const results = ActorQueryOperation.getSafeBindings(await this.mediatorQueryOperation.mediate({
-        context,
-        operation: ActorAbstractPath.FACTORY.createPath(path.subject, predicate, blankNode, path.graph),
-      }));
-      const bindingsStream = results.bindingsStream.transform<Bindings>({
-        filter: item => item.get(bString).equals(path.object),
-        transform(item, next, push) {
-          push(Bindings({ }));
-          next();
-        },
-      });
-      return { type: 'bindings', bindingsStream, variables: []};
     }
+    // If (!sVar && !oVar)
+    const blankNode = this.generateVariable();
+    const bString = termToString(blankNode);
+    const results = ActorQueryOperation.getSafeBindings(await this.mediatorQueryOperation.mediate({
+      context,
+      operation: ActorAbstractPath.FACTORY.createPath(path.subject, predicate, blankNode, path.graph),
+    }));
+    const bindingsStream = results.bindingsStream.transform<Bindings>({
+      filter: item => item.get(bString).equals(path.object),
+      transform(item, next, push) {
+        push(Bindings({ }));
+        next();
+      },
+    });
+    return { type: 'bindings', bindingsStream, variables: []};
```

Comment:
`variables` is empty, I suspect this is wrong. But it looks like this used to be like this before as well. Was this intentional @joachimvh?

Reply:
It's a path where both object and subject are not variables (see line 121) something like :a :p+ :b, so no variables there. Not sure what this means semantically

Reply:
This is actually correct since this corresponds to the cases were both ends are not variables, so something like `:s :p+ :o.`. This will result in a potential empty binding.

Reply:
Ah, I see, thanks for clarifying!

Reply:
So semantically it is pretty much a check to see if this pattern exists at the endpoint.